### ISSUE TITLE
Add ability to check the data in check_store_contains as in the order that was displayed in the UI

### ISSUE THREAD

**Issue by psuryateja123** [westy/pyseext#15]
It would be nice to check the data that will passed into check_store_contains as in the order that was displayed in the UI, So would expect the data that was passed into need to match the order that was displayed in the UI.

This is a use case for checking view history, as certain actions needed to happen in an order or sequence.

**Comment by westy** [comment]
Yeah, sounds like an option to check the order of the items in the store match that passed into the method.
It's not really related to the UI from the pyseext point of view, but yeah, the interface could be expanded to do this.

I'll try and get to it.

Cheers

**Comment by psuryateja123** [comment]
Thanks Westy
